### DISCUSSION TITLE
Feature request: Css variable in Layout component not present in autocomplete of child

### DISCUSSION THREAD

**Discussion by nojaf** [withastro/language-tools#942] [Feature requests]
### Describe the Bug

When I define CSS variables in a Layout or page

```html
<style>
	:root {
		--yellow: #FF0;
		--red: #F00;
	}
</style>
```

I don't get autocomplete for these in my component files.

### Steps to Reproduce

1. `npm init astro` using template <NAME>
2. Create Layout with CSS variable
3. Try and consume in child component
4. No autocomplete

### Link to Minimal Reproducible Example

https://github.com/nojaf/astro-variable-repro

**Comment by Princesseuh** [comment]
This is not a bug, the feature just isn't there.
